src/nvim/window.c

```diff
@@ -6945,15 +6945,18 @@ char *file_name_in_line(char *line, int col, int options, int count, char *rel_f
   }
 
   // Search forward for the last char of the file name.
-  // Also allow ":/" when ':' is not in 'isfname'.
-  len = 0;
+  // Also allow ":/" and ":\" when ':' is not in 'isfname'.
```

Comment:
"/" is usually, internally, understood to mean "any valid path separate" in most contexts. Is the comment change actually necessary?

Reply:
I think the comment is there to clarify about the logic that considers ':' even if not in 'isfname'. I thought that it might be useful to add that ':\' also is checked, but if we want to be very explicit all ':\', ':/', ':?' and ':#' are actually considered on `path_has_drive_letter` function.

Reply:
Note that this comment was originally referring to URLs.

Reply:
Yes but the logic was exclusively for URLs, now it also considers drives